wgengine/wgengine.go

```diff
@@ -89,10 +89,12 @@ type Engine interface {
 	// new Engine.
 	Close()
 
-	// Wait waits until the Engine's Close method is called or the
-	// engine aborts with an error. You don't have to call this.
+	// Wait returns a channel that is closed when the Engine's
+	// Close method is called, the engine aborts with an error,
+	// or it shuts down due to the closure of the underlying device.
+	// You don't have to call this.
 	// TODO: return an error?
```

Comment:
Delete this TODO now.

If we're copying the context.Context pattern, the Err method would be its own accessor after Done is cosed.